src/main/java/com/google/sps/servlets/GroceryServlet.java

```diff
@@ -107,27 +110,37 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     }
 
     // creates arraylist and starts query
-    ArrayList<String> groceryList = new ArrayList<String>();
+    ArrayList<Grocery> groceryList = new ArrayList<>();
     long familyID = (long) userInfoEntity.getProperty(FAMILY_ID);
     Query groceryQuery = new Query(GROCERY)
       .setFilter(new Query.FilterPredicate(FAMILY_ID, Query.FilterOperator.EQUAL, familyID));   
     PreparedQuery familyGrocery= datastore.prepare(groceryQuery);
-    groceryList = checkGroceries(familyGrocery);   
+    groceryList = checkGroceries(familyGrocery, userEmail);   
 
     Gson gson = new Gson();
     response.setContentType("application/json;");
     response.getWriter().println(gson.toJson(groceryList));
     }
 
-  // returns items from the query that match the users familyID
-  private ArrayList<String> checkGroceries(PreparedQuery familyGrocery) {
-    ArrayList<String> groceryList = new ArrayList<String>();
+  // returns items from the query that match the users familyID in the Grocery object    
+  private ArrayList<Grocery> checkGroceries(PreparedQuery familyGrocery, String userEmail) {
+    boolean match;
+    long timestamp = System.currentTimeMillis();
+    System.out.println(timestamp);
```

Comment:
why do you need the timestamp?

Reply:
It's not currently being used but I was considering ordering the list chronologically